torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -591,6 +592,13 @@ ProcessGroupNCCL::ProcessGroupNCCL(
             << "\nUSE_HIGH_PRIORITY_STREAM: "
             << options_->is_high_priority_stream
             << "\nNCCL_DEBUG: " << ncclDebugLevel;
+
+  ucc_lib = loadTorchUCC();
```

Comment:
Shouldn't this be done statically only once? Do we need to load the library each time we create a new ProcessGroup?

Reply:
I agree. Fixed.